index.html

```diff
@@ -402,6 +408,56 @@ <h2 class="title-bar-item">JSON Path Finder</h2>
   document.querySelectorAll(".big-box").forEach((el) => { el.style.height = (window.innerHeight - 90) + "px" });
 }
 
+// set initial dark or light mode based on users preference
+if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
+    darkMode();
+} else {
+    lightMode();
+}
+
+// watch for user to change system to or from dark mode
+window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', event => {
+    const newColorScheme = event.matches ? "dark" : "light";
+    if (newColorScheme == "light") lightMode();
+    if (newColorScheme == "dark") darkMode();
+});
+
+function invert() { // dark mode
+  let css = 'html {-webkit-filter: invert(100%) brightness(.37) saturate(2.8) hue-rotate(180deg);' +
```

Comment:
I think we need a more accessible contrast and I'd prefer a softer black. Fiddling around, this feels nice:
```suggestion
  let css = 'html {-webkit-filter: invert(85%) brightness(.90) saturate(1.5) hue-rotate(180deg);' +
```